frame/contracts/proc-macro/src/lib.rs

```diff
@@ -64,7 +64,6 @@ fn derive_debug(
 	#[cfg(not(feature = "full"))]
 	let fields = {
 		drop(fmt);
-		drop(data);
```

Comment:
Why? This was put there because otherwise we get a dead_code warning because it is unused. I assume nothing about that changed?

Reply:
clippy error: calls to std::mem::drop with a reference instead of an owned value. Dropping a reference does nothing.

Reply:
If you don't need to change that, please close it